acai-ios/CollectionViewCells/QuantitySelectionCollectionViewCell.swift

```diff
@@ -88,38 +95,27 @@ class QuantitySelectionCollectionViewCell: UICollectionViewCell {
     }
     
     @objc func incrementQuantity() {
-        if let object = object {
-            self.quantity = quantity + 1
-            updateQuantityLabelText(object: object)
-        }
+        delegate?.valueIncremented()
     }
     
     @objc func decrementQuantity() {
-        if let object = object, quantity > 1 {
-            self.quantity = quantity - 1
-            updateQuantityLabelText(object: object)
-        }
+        delegate?.valueDecremented()
     }
-    
-    func updateQuantityLabelText(object: Any) {
-        if object is BowlItem {
-            quantityLabel.text = "\(quantity) Bowl"
-        }
-        if let text = quantityLabel.text, quantity != 1 {
-            quantityLabel.text = "\(text)s"
+
+    func configure(for quantity: NSNumber, and itemType: MenuItem.ItemType) {
```

Comment:
any reason why quantity is an `NSNumber` instead of just `Int`?

Reply:
NSNumber conforms to ListDiffable